components/MovieItem/MovieItem.tsx

```diff
@@ -2,33 +2,44 @@ import { FC } from "react";
 import Image from "next/image";
 
 import { HiMiniStar } from "react-icons/hi2";
+import Link from "next/link";
 
 interface MovieItemTypeProp {
-  id?: string;
-  title?: string;
-  rating?: string;
+  imdbid: string;
+  title: string;
+  imgURL: string;
+  rating: string;
 }
 
-const MovieItem: FC<MovieItemTypeProp> = ({ id, title, rating }) => {
-  const url = "https://picsum.photos/800/600";
-  if (url === null) return;
+const MovieItem: FC<MovieItemTypeProp> = ({
+  imdbid,
+  title,
+  imgURL,
+  rating,
+}) => {
+  if (imgURL === null) return;
   return (
     <div>
-      <div className="relative transition duration-300 ease-out hover:scale-105">
+      <Link
+        href={`/movie/${imdbid}`}
+        className="relative block transition duration-300 ease-out hover:scale-105 max-w-sm"
+      >
         <Image
-          src={url}
+          src={imgURL}
           width="0"
           height="0"
           sizes="100vw"
           className="w-full cursor-pointer rounded-lg"
-          alt={""}
+          alt={imdbid}
```

Comment:
question: Do you see any problem with adding as description to this image an id, let's say `alt="tt0108052"`?

<details>
<summary>Answer:</summary>

This alt is read by SR (Screen Readers) to users. The id is useless (and annoying). For that reason, it's better to have an empty alt, so that the SR skips the image.
</details>